crates/rolldown/src/chunk/mod.rs

```diff
@@ -155,7 +155,10 @@ impl Chunk {
           Ok(())
         },
       )?;
-
+    // add banner
+    if let Some(banner) = &output_options.banner {
+      concat_source.prepend_source(Box::new(RawSource::new(banner.to_string())))
+    }
```

Comment:
Here `prepend_source` is unnecessary, you can move it to the first of chunk `concat_source` and using `addSource`.

Reply:
I'm not good at `rust` at all, so maybe U should review carefully +_+